app/src/main/res/values/strings.xml

```diff
@@ -4,136 +4,137 @@
     <string name="app_base_url">https://ntfy.sh</string> <!-- If changed, you must also change google-services.json! -->
 
     <!-- Notification channels -->
-    <string name="channel_notifications_min_name">Notifications (Min Priority)</string>
-    <string name="channel_notifications_low_name">Notifications (Low Priority)</string>
-    <string name="channel_notifications_default_name">Notifications (Default Priority)</string>
-    <string name="channel_notifications_high_name">Notifications (High Priority)</string>
-    <string name="channel_notifications_max_name">Notifications (Max Priority)</string>
+    <string name="channel_notifications_min_name">Notifications (min priority)</string>
+    <string name="channel_notifications_low_name">Notifications (low priority)</string>
+    <string name="channel_notifications_default_name">Notifications (default priority)</string>
+    <string name="channel_notifications_high_name">Notifications (high priority)</string>
+    <string name="channel_notifications_max_name">Notifications (max priority)</string>
     <string name="channel_subscriber_service_name">Subscription Service</string>
     <string name="channel_subscriber_notification_title">Listening for incoming notifications</string>
-    <string name="channel_subscriber_notification_instant_text">You are subscribed to instant delivery topics</string>
-    <string name="channel_subscriber_notification_instant_text_one">You are subscribed to one instant delivery topic</string>
-    <string name="channel_subscriber_notification_instant_text_two">You are subscribed to two instant delivery topics</string>
-    <string name="channel_subscriber_notification_instant_text_three">You are subscribed to three instant delivery topics</string>
-    <string name="channel_subscriber_notification_instant_text_four">You are subscribed to four instant delivery topics</string>
-    <string name="channel_subscriber_notification_instant_text_more">You are subscribed to %1$d instant delivery topics</string>
-    <string name="channel_subscriber_notification_noinstant_text">You are subscribed to topics</string>
-    <string name="channel_subscriber_notification_noinstant_text_one">You are subscribed to one topic</string>
-    <string name="channel_subscriber_notification_noinstant_text_two">You are subscribed to two topics</string>
-    <string name="channel_subscriber_notification_noinstant_text_three">You are subscribed to three topics</string>
-    <string name="channel_subscriber_notification_noinstant_text_four">You are subscribed to four topics</string>
-    <string name="channel_subscriber_notification_noinstant_text_more">You are subscribed to %1$d topics</string>
+    <string name="channel_subscriber_notification_instant_text">Subscribing to instant delivery topics</string>
```

Comment:
Could maybe do an "en" branch to translate from, and then en_US and en_UK specifically to avoid that as a baseline. Otherwise (and I argue anyhow) a needless "you" is invoked, which adds complexity.
Not a big deal, but that is my idea anyhow.
Visually speaking more text induces stress, and it adds a lot of time in navigation when using a screen reader.